### PR TITLE
Add more validations for mobile.

### PR DESCRIPTION
**Pull request by sfdctaka**
Adding more validations specifically for developing LWC components to be more compatible on mobile devices:
* Form elements require labels.
* Clickable icons require labels.
* Font size need to be bigger than 14px.
* Text should not be truncated or wrapped.

### CODE REVIEW

**Comment by sfdctaka** [comment]
@billyma @ambreeen

**Comment by sfdctaka** [comment]
@maliroteh-sf @khawkins

File: core/src/test/java/com/salesforce/slds/validation/validators/MobileValidationTest.java [hunk 1]
```diff
@@ -1,10 +1,3 @@
-/*
- * Copyright (c) 2018, salesforce.com, inc.
- * All rights reserved.
- * SPDX-License-Identifier: BSD-3-Clause
- * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
```
Comment (billyma) [inline core/src/test/java/com/salesforce/slds/validation/validators/MobileValidationTest.java:6]: Would you mind adding this header back to the file? Thanks.

File: lsp/src/main/java/com/salesforce/slds/lsp/codeactions/CodeActionConverter.java [hunk 2]
```diff
@@ -59,8 +59,9 @@
                         return createAlternativeTokenCodeAction(params, info.get());
                     case UTILITY_TOKENS:
                         return createUtilityTokenCodeAction(params, info.get());
-                    case NON_MOBILE_FRIENDLY:
-                        return createNonMobileFriendlyBaseComponentCodeAction(params, info.get());
+                    case MOBILE_SLDS:
```
Comment (billyma) [inline lsp/src/main/java/com/salesforce/slds/lsp/codeactions/CodeActionConverter.java:62]: Just confirm, we will update this before updating the client. Is that correct?

**Review by billyma — APPROVED** [review]
Thanks @sfdctaka - can you verify the files contains the Open Source header?

```java
/*
 * Copyright (c) 2018, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: BSD-3-Clause
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
```

File: lsp/src/main/java/com/salesforce/slds/lsp/codeactions/CodeActionConverter.java (hunk 2 above)
Comment (maliroteh-sf) [reply lsp/src/main/java/com/salesforce/slds/lsp/codeactions/CodeActionConverter.java:62]: @billyma In my PR I am removing this comment and adding code actions for suppressing these warnings via the Quick Fix menu (if that's what you're asking about).

Comment (billyma) [reply lsp/src/main/java/com/salesforce/slds/lsp/codeactions/CodeActionConverter.java:62]: Yes, just want to confirm your change will be in before we update the client. So, the experience won't be weird for the user.

File: core/src/main/resources/tokens/slds/components.json [hunk 3]
```diff
@@ -3,7 +3,8 @@
   "annotations" : {
     "base" : "true",
     "name" : "accordion",
-    "role" : ".slds-accordion",
+    "role" : null,
+    "lwc" : "true",
```
Comment (ambreeen) [inline core/src/main/resources/tokens/slds/components.json:7]: I wanted to double check, are these json file updates for 232 or 234?

Comment (sfdctaka) [reply core/src/main/resources/tokens/slds/components.json:7]: They are for 232.

Comment (ambreeen) [reply core/src/main/resources/tokens/slds/components.json:7]: Ok great, thanks @sfdctaka !

**Review by ambreeen — APPROVED** [review]
looks good!

**Comment by sfdctaka** [comment]
@billyma @ambreeen Comment headers are now in the files.

**Pull request merged**